iot-hub/Samples/device/X509DeviceCertWithChainSample/Program.cs

```diff
@@ -47,7 +47,6 @@ public static async Task<int> Main(string[] args)
 
             var sample = new X509DeviceCertWithChainSample(deviceClient);
             await sample.RunSampleAsync();
-            await deviceClient.CloseAsync();
```

Comment:
why was this removed?

Reply:
It is already there in the sample run, not program.cs. I think someone added it here again and the sample was failing as we were calling Close twice.